public_dropin_notebook_environments/r_notebook/agent/README.md

```diff
@@ -0,0 +1,13 @@
+# Kernel Agent
```

Comment:
Do we need to remove this README or modify as it contains terminology that is specific to DR? @mx-iao

Reply:
@k-radhak is this agent folder required to be present in the user's custom environments in order to have a DR Notebook-compatible environment?
If so, we should keep the README.me but instead of it having DataRobot internal information we should modify it to just say something along the lines of "This folder contains dependencies required to use this custom environment with DataRobot Notebooks. Please do not modify or delete this folder from your Docker context."

Reply:
and we should do the same for the other agent/README.md files in the other two templates as well

Reply:
The agent folder is required I will update the readme.